build-support/docker/Build-UI.dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG ALPINE_VERSION=3.13
```

Comment:
Sorry I couldn't help but lurk, it looks like we probably want to bump to 3.14 for all Dockerfiles since 3.13 has CVEs

Reply:
More eyes always appreciated 👍